I realize this manual page is generated from XML source..  The source has a
markup bug ner the logfile option that causes a stray PP to be generated.

--- rotatelogs.8-unpatched	2016-03-07 10:39:11.869723628 -0500
+++ rotatelogs.8	2016-03-07 10:39:37.893645102 -0500
@@ -68,7 +66,23 @@
 Use a circular list of filenames without timestamps\&. With -n 3, the series of log files opened would be "logfile", "logfile\&.1", "logfile\&.2", then overwriting "logfile"\&. Available in 2\&.4\&.5 and later\&.  
 .TP
 \fIlogfile\fR
-.PP The path plus basename of the logfile\&. If \fIlogfile\fR includes any '%' characters, it is treated as a format string for strftime(3)\&. Otherwise, the suffix \fI\&.nnnnnnnnnn\fR is automatically added and is the time in seconds (unless the -t option is used)\&. Both formats compute the start time from the beginning of the current period\&. For example, if a rotation time of 86400 is specified, the hour, minute, and second fields created from the strftime(3) format will all be zero, referring to the beginning of the current 24-hour period (midnight)\&. .PP When using strftime(3) filename formatting, be sure the log file format has enough granularity to produce a different file name each time the logs are rotated\&. Otherwise rotation will overwrite the same file instead of starting a new one\&. For example, if \fIlogfile\fR was /var/log/errorlog\&.%Y-%m-%d with log rotation at 5 megabytes, but 5 megabytes was reached twice in the same day, the same log file name would be produced and log rotation would keep writing to the same file\&.  
+The path plus basename of the logfile\&. If \fIlogfile\fR includes
+any '%' characters, it is treated as a format string for
+strftime(3)\&. Otherwise, the suffix \fI\&.nnnnnnnnnn\fR is
+automatically added and is the time in seconds (unless the -t option
+is used)\&. Both formats compute the start time from the beginning of
+the current period\&. For example, if a rotation time of 86400 is
+specified, the hour, minute, and second fields created from the
+strftime(3) format will all be zero, referring to the beginning of the
+current 24-hour period (midnight)\&. .PP When using strftime(3)
+filename formatting, be sure the log file format has enough
+granularity to produce a different file name each time the logs are
+rotated\&. Otherwise rotation will overwrite the same file instead of
+starting a new one\&. For example, if \fIlogfile\fR was
+/var/log/errorlog\&.%Y-%m-%d with log rotation at 5 megabytes, but 5
+megabytes was reached twice in the same day, the same log file name
+would be produced and log rotation would keep writing to the same
+file\&.
 .TP
 \fIrotationtime\fR
 The time between log file rotations in seconds\&. The rotation occurs at the beginning of this interval\&. For example, if the rotation time is 3600, the log file will be rotated at the beginning of every hour; if the rotation time is 86400, the log file will be rotated every night at midnight\&. (If no data is logged during an interval, no file will be created\&.)  
